src/AspectMock/Intercept/BeforeMockTransformer.php

```diff
@@ -11,7 +11,7 @@ class BeforeMockTransformer extends WeavingTransformer
     protected $before = " if ((\$__am_res = __amock_before(\$this, __CLASS__, __FUNCTION__, array(%s), false)) !== __AM_CONTINUE__) return \$__am_res; ";
     protected $beforeStatic = " if ((\$__am_res = __amock_before(get_called_class(), __CLASS__, __FUNCTION__, array(%s), true)) !== __AM_CONTINUE__) return \$__am_res; ";
 
-    public function transform(StreamMetaData $metadata)
+    public function transform(StreamMetaData $metadata): string
```

Comment:
This kind of changes break compatibility with older versions, so the lowest supported version must be changed from 2.2.0 to 2.3.0.

Reply:
Alright, this sounds reasonable. What should we do to have it released under `2.3.0`?

Reply:
2.3.0 of AspectMock?
I created v2 branch based on 2.2.0 release, please make another pull request to it.